### DISCUSSION TITLE
error installing @budibase/cli on debian latest

### DISCUSSION THREAD

**Discussion by poirazis** [Budibase/budibase#7108] [Help] [answered]
Hey!
I have a debian 11 server already running BB , and i tried to reinstall @budibase/cli via npm 
I am getting the below error , is this BB or something else ?

<img width="1009" alt="Screenshot 2022-08-05 at 11 54 38 AM" src="https://user-images.githubusercontent.com/80512170/183041759-6de12dce-96e3-431b-9b24-f4a9d4b139fa.png">

**Comment by melohagan** [comment]
Hey @poirazis 

Which version of node are you running? Make sure you are using either v14 or v16

**Comment by poirazis** [comment, marked as answer]
Hey @melohagan 
Using node 16 , same error. anything else i should check ?
<img width="542" alt="Screenshot 2022-08-08 at 3 25 23 PM" src="https://user-images.githubusercontent.com/80512170/183417505-dc40eb9e-ade9-4af8-ba84-7d785dd1bf40.png">

**Reply by mike12345567** [reply]
Hi @poirazis - I see you are executing these commands as `root` - `node-gyp` isn't particularly a fan of running in the root user but you can get around it: https://timjrobinson.com/fixing-node-gyp-permission-denied-when-running-as-root/

Hopefully this helps!

**Reply by poirazis** [reply]
Thank you both @melohagan and @mike12345567 
🙏
